PeakLocalisations/Localisation/Blocks/ConfigureGoal.xcstrings

```diff
@@ -66,6 +84,12 @@
             "state" : "translated",
             "value" : "Build"
           }
+        },
+        "es" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Construir"
```

Comment:
this is the verb to build, unsure if its meant to be

Reply:
Ah yep, that's the intention